blast/README.md

```diff
@@ -70,3 +70,15 @@ dot -Tpng blast.dot > blast.png
 display blast.png
 ```
 
+Additionally, you can generate random data to adjust the total runtime:
+```
+./blast_data_gen 100 500 > test.fasta
+./makeflow_blast -d nt -i test.fasta -p blastn --num_seq 5 --makeflow blast_test.mf
+makeflow blast_test.mf
+```
+
+The number and length of sequences can be adjusted for your needs, with the first number 
+adjusting the number of contigs and the second adjusting the length of these contigs.
+`blast_data_gen.pl` produces contigs containing random AGCT sequences.
+
+The provided values produces a workflow that runs in ~5 minutes.
```

Comment:
Clarify if this means a single local machine.

Can you provide a configuration for a large workflow that makes sense to run in parallel?
(We need to provide a milestone for others to compare against.)